server/utils/scraper.ts

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
+import * as cheerio from 'cheerio';
+
+export function scraper(body: any) {
+  const obj = {
+    title: 'title.title',
+    description: 'desc.desc',
+    image: 'image.image',
+  };
+  return obj;
+}
+
+interface Metadata {
+  title: string;
+  description: string;
+  image: string;
+}
+// mimic a browser request by setting headers
+const headers = {
+  'User-Agent':
+    'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36',
+  'Accept-Language': 'en-US,en;q=0.9',
+  'Accept-Encoding': 'gzip, deflate, br',
+  Connection: 'keep-alive',
+};
+
+export async function getHtml(urls: string[]) {
```

Comment:
1. missing return type
2. how does it help the client that sent the request that you `console.log(error)` the error? 
3. consider storing the css-selectors of each html element you want to get it's text in a way that is more maintainable